TAC/Meetings/2023/2023-04-06/CCC-TAC-Minutes-2023-04-06.md

```diff
@@ -0,0 +1,141 @@
+* DM welcomed the members of the TAC and reviewed the values of the CCC and the antitrust policy of the Linux Foundation
+* Kurt Taylor (KT) assisted with the meeting minutes
+* DM reviewed the agenda
+
+### Roll Call / Attendance
+
+***Voting members of the TAC***
+
+*Per the [charter](https://charter.confidentialcomputing.io), all [CCC Premier members](https://confidentialcomputing.io/members/) receive one vote on the TAC. Quorum for votes is at least 50% of voting members present.*
+
+* Accenture - Giuseppe Giordano 
+* Arm - Thomas Fossati  / Michael Lu
+* Google - Cfir Cohen  / Catalin Sandu
+* Huawei - Zhipeng (Howard) Huang + 
+* Intel - Dan Middleton (TAC chair) + / Simon Johnson
+* Meta - Eric Northup / Shankaran Gnanashanmugam
+* Microsoft - Dave Thaler + 
+* Red Hat - Lily Sturmann  + / Yash Mankad +
+
+   " + " indicates attending
+
+***Other Attendees***
+
+* Alec Fernandez (Microsoft)
+* Fan Zhang (Yale)
+* Alan Czeszynski (BeeKeeper AI)
+* Hongbo Chen (Indiana)
+* Tiziano Santoro (Google)
+* Mark Novak (JP Morgan Chase)
+* Drasko Draskovic (Ultraviolet)
+* Ofir Azoulay-Rozanes (Anjuna)
+* Leonardo Garcia (Linaro)
+* Xinxin Fan (Iotex)
+* Mona Vij (Intel)
+* Vidhya Krishnan (NVIDIA)
+* Xiaofeng Wang (Indiana)
+* Ahmad Atamli (NVIDIA)
+* Kurt Taylor (LF)
+* Helen Lau (LF)
+
+
+**Meeting Minute Approval**
+
+Proposed: That the minutes of the March 23, 2023 meeting of the Technical Advisory Council of the Confidential Computing Consortium as distributed to the members of the TAC in advance of this meeting are hereby adopted and approved.
+
+* Quorum was reached and the minutes were approved by the voting members of the TAC.
+
+
+**Action Item Review**
+
+* DM reviewed a subset of the action items
+* KT discussed the staus of the white paper conversion for source control
+  * Dave Thaler (DT) found the original source for the [Technical Analysis white paper](https://docs.google.com/document/d/17PhrIXvFJsAIJryYjpezmfdSl1BSB1x1tE2FTbriHyc/edit?pli=1)
+* Alec Fernandez (AF) discussed the blog post state and review process
+
+  ***ACTION*** Helen to ask about turning on the comment feature on the website for blog comments
+
+**2023 Goals**
+
+* DM addressed 2023 [plans](https://docs.google.com/document/d/1BLsI0hv9ybHl-FBNqHp6bJzy6ng8yKs__556bTqBswc/edit)
+* Project Progression
+  * Howard Huang (HH) discussed the [PR #160](https://github.com/confidential-computing/governance/pull/160) for refining the definition of the project progression policy
+    * Need to clarify and seperate the benefits for each state
+    * DT proposed that the TAC remove the 2 later unapproved states originally suggested but not approved
+    * Headed home suggested that there be more benefits for the sandbox stage
+* HH mentioned a new project [Bao](https://github.com/bao-project) and asked the TSC chair to speak at a future TAC meeting
+
+
+**TAC Tech Talk**
+* Towards Trust Minimization in TEEs and Applications - Fan Zhang (CDCC)
+
```

Comment:
Was there no discussion worth noting?

Reply:
I think this is fine given the work is not published yet. The community will be able to read the publication soon enough.